mqtt 3/1.0.0/modules/ROOT/pages/mqtt3-connector-studio-configuration.adoc

```diff
@@ -0,0 +1,412 @@
+* *WS*
+* *WSS*
+
+[[configure-authentication]]
+== Configure Credentials For Authentication
+
+Authentication credentials are optional, but you can provide a username and a password if it is required.
+
+In the following example, you configure the authentication by providing a basic username and password:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Click *Create*.
+. In the filter box type `mqtt` and select *MQTT3 Config*.
+. Click *OK*.
+. In the *MQTT3 Config* window, for *Connection*, select *MQTT3 URL Connection*.
+. Set the following fields:
++
+* *Client ID*: `smart-bentley-123`.
+* *Username*: `username`.
+* *Password*: `passtest`.
+* *URL*: `tcp://127.0.0.1:1883"`.
+[start=7]
+. Click *OK*.
+
+In the XML editor, the  `<mqtt3:connection>`, `username` and `password` configurations look like this:
+
+[source,xml,linenums]
+----
+<mqtt3:config name="MQTT_Config">
+    <mqtt3:connection clientId="smart-bentley-123" username="usertest" password="passtest" url="tcp://127.0.0.1:1883"/>
+</mqtt3:config>
+----
+
+You can also provide a client certificate to authenticate the connection by setting a TLS context.
+In the following example, you configure the TLS context:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Select your MQTT3 Config and click *Edit*.
+. In the *MQTT3 Config*, click *SSL/TLS*.
+. For *TLS Context* select *Edit inline*.
+. In *Trust Store Configuration*, set the following fields:
++
+* *Path*: `tls/truststore.jks`.
+* *Password*: `racing`.
+* *Type*: `jks"`.
+[start=6]
+. Click *OK*.
+
+.MQTT3 TLS Context configuration
+image::mqtt3-tls-configuration.png[MQTT3 TLS Context configuration in Global Element Properties window]
+
+In the XML editor, the `<tls:context>`,`path`, `password` and `type` configurations look like these:
+
+[source,xml,linenums]
+----
+<mqtt3:config name="MQTT_TLS_Config">
+    <mqtt3:connection clientId="smart-bentley-tls-123" username="usertest" password="passtest" url="ssl://localhost:8883" >
+        <tls:context>
+            <tls:trust-store path="tls/truststore.jks" password="racing" type="jks"/>
+        </tls:context>
+    </mqtt3:connection>
+</mqtt3:config>
+----
+
+[[configure-failover]]
+== Configure a Fail-Over Server List
+
+There are certain deployment schemas that consist of multiple brokers working together in order to provide clients with several connection endpoints. When there is more than one available server that the client can connect to, there are two possible scenarios: either each MQTT server operates separately or they might work together and share a state (cluster mode), in which case, you might want to specify how the MQTT client behaves in the event of a reconnection.
+
+When you provide a failover server list, the connector can iterate over the list until it successfully establishes a connection with one of the provided endpoints.
+
+In the following example, you configure the fail-over server list for an *MQTT3 Fail-Over Connection*:
```

Comment:
You use both fail-over and failover. I believe the latter is correct. S&R for consistency